### DISCUSSION TITLE
ABACUS+DeePMD requires MPI parallelism

### DISCUSSION THREAD

**Discussion by YuLiu98** [deepmodeling/abacus-develop#5272] [General]
### Details

Currently, ABACUS enables molecular dynamics calculations with DeePMD-kit. However, the MPI parallelism does not work, which affects the performance.

The MPI parallelism should be supported in the future.

### Task list for Issue attackers (only for developers)

- [ ] Reproduce the performance issue on a similar system or environment.
- [ ] Identify the specific section of the code causing the performance issue.
- [ ] Investigate the issue and determine the root cause.
- [ ] Research best practices and potential solutions for the identified performance issue.
- [ ] Implement the chosen solution to address the performance issue.
- [ ] Test the implemented solution to ensure it improves performance without introducing new issues.
- [ ] Optimize the solution if necessary, considering trade-offs between performance and other factors (e.g., code complexity, readability, maintainability).
- [ ] Review and incorporate any relevant feedback from users or developers.
- [ ] Merge the improved solution into the main codebase and notify the issue reporter.

**Comment by QuantumMisaka** [comment]
In ASE and other scripts using deepmd (even deepmd-kit itself ?) are using OMP to do parallelism, I consider that MPI will be a little hard, but it is meaningful.

**Comment by WHUweiqingzhou** [comment]
Looking forward to it.
